data/json/professions.json

```diff
@@ -3438,6 +3541,39 @@
       "female": { "entries": [ { "item": "bra" }, { "item": "panties" } ] }
     }
   },
+  {
+    "type": "profession",
+    "id": "tech_maint",
+    "name": "Maintenance Technician",
+    "description": "From heat exchangers to washing machines; if it exists, you've probably gotten your hands on it at least once, working behind the scenes to ensure that the things everyone took for granted worked smoothly and properly.  It barely matters to you how and who screwed up this time, you will get it back in working order in no time.",
+    "points": 5,
+    "proficiencies": [ "prof_plumbing", "prof_elec_soldering", "prof_metalworking", "prof_appliance_repair" ],
+    "skills": [
+      { "level": 5, "name": "fabrication" },
+      { "level": 5, "name": "electronics" },
+      { "level": 5, "name": "mechanics" },
+      { "level": 5, "name": "traps" }
```

Comment:
Level 5 proficiency in traps probably should be reserved to combat engineers and the like.
Representing the ability to improvise traps from mechanical knowledge would be something like a level 3, I think.
(The crafting recipes for blade traps and booby traps are lv3, actual landmine is lv5.)